packages/remix-dev/cli/migrate/questions.ts

```diff
@@ -1,29 +1,29 @@
 import inquirer from "inquirer";
 
-import { transformOptions } from "./transform-options";
+import { migrationOptions } from "./migration-options";
 
 export type Answers = {
   projectDir: string;
-  transform: string;
+  migration: string;
 };
 
 type QuestionsArgs = {
-  input: { projectDir: string; transform: string };
+  input: { projectDir: string; migration?: string };
```

Comment:
Should have done this in my PR, but `projectDir` is optional too.

```suggestion
  input: { projectDir?: string; migration?: string };
```

Reply:
Actually, `meow` will not give `undefined` for `projectDir` when it is omitted, but rather `""`. Or at least that's what the types in my editor indicate.